docs/UserGuide.md

```diff
@@ -114,20 +114,32 @@ Displays window showing the list of commands and a clickable URL to access help
 
 Format: `help`
 
+Shortcut Key: `F1`
+
 <div style="page-break-after: always;"></div>
 
 #### Undoing a change : `undo`
 
 Undoes the last change made to SOCket.
 
 Format: `undo`
+
+Shortcut Key:
+1. `Ctrl+Z` for __Window user__.
+2. `Command+Z` for __Mac user__.
+
 * A message is shown if no changes were made to SOCket.
 
 #### Redoing an undone change : `redo`
 
 Restores a previously undone change made to SOCket.
 
 Format: `redo`
+
+Shortcut Key: 
+1. `Ctrl+Shift+Z` for __Window user__.
+2. `Command+Shift+Z` for __Mac user__.
```

Comment:
Window should be Windows and can consider changing Mac to MacOS or whatever the appropriate OS name is for mac users